Sources/App/Commands/Analyzer.swift

```diff
@@ -87,6 +87,7 @@ func analyze(application: Application, packages: EventLoopFuture<[Package]>) ->
         .flatMap { RecentPackage.refresh(on: application.db) }
         .flatMap { RecentRelease.refresh(on: application.db) }
         .flatMap { Search.refresh(on: application.db) }
+        .flatMap { Stats.refresh(on: application.db) }
```

Comment:
How much of a load do these refreshes put on the box? I know we've got *loads* of headroom right now, and I know that's a hard question to answer (I don't expect an answer) but it's worth keeping an eye on.